compiler/noirc_frontend/src/lexer/lexer.rs

```diff
@@ -682,12 +746,32 @@ impl<'a> Lexer<'a> {
         c == '\t' || c == '\n' || c == '\r' || c == ' '
     }
 
+    fn is_fv_attribute(attribute_name: &str) -> bool {
+        attribute_name == "requires" || attribute_name == "ensures"
+    }
+
     /// Skips white space. They are not significant in the source language
     fn eat_whitespace(&mut self, initial_char: char) -> SpannedToken {
         let start = self.position;
         let whitespace = self.eat_while(initial_char.into(), Self::is_code_whitespace);
         SpannedToken::new(Token::Whitespace(whitespace), Span::inclusive(start, self.position))
     }
+
+    /// Skips white space and backslashes. After a backslash they are not significant in the annotation body
+    fn eat_whitespace_and_backslash(&mut self, initial_char: char) -> SpannedToken {
+        let start = self.position;
+        let is_whitespace_or_backslash = |x: char| -> bool {x == '\\' || Self::is_code_whitespace(x)};
+        let whitespace = self.eat_while(initial_char.into(), is_whitespace_or_backslash);
+        SpannedToken::new(Token::Whitespace(whitespace), Span::inclusive(start, self.position))
```

Comment:
So you will create a "whitespace" token that contains the backslash 🤔 🤔 🤔 

I guess it's fine... for a prototype. I just wouldn't call the backslash whitespace.